### PR TITLE
Add precache image to TALO CI

### PR DESCRIPTION
**Pull request by vitus133**
Add the pre-caching workload image to
topology-aware lifecycle operator (TALO) build
/cc @nishant-parekh @irinamihai @danielmellado

### CODE REVIEW

File: core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay [hunk 1]
```diff
@@ -57,3 +57,4 @@ registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator-bundle qu
 registry.ci.openshift.org/ocp-kni/4.11:cluster-group-upgrades-operator quay.io/openshift-kni/cluster-group-upgrades-operator:4.11.0 quay.io/openshift-kni/cluster-group-upgrades-operator:latest
 registry.ci.openshift.org/ocp-kni/4.11:cluster-group-upgrades-operator-bundle quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:4.11.0 quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:latest
 registry.ci.openshift.org/ocp-kni/4.11:cluster-group-upgrades-operator-recovery quay.io/openshift-kni/cluster-group-upgrades-operator-recovery:4.11.0 quay.io/openshift-kni/cluster-group-upgrades-operator-recovery:latest
+registry.ci.openshift.org/ocp-kni/4.11:cluster-group-upgrades-operator-recovery quay.io/openshift-kni/cluster-group-upgrades-operator-precache:4.11.0 quay.io/openshift-kni/cluster-group-upgrades-operator-precache:latest
```
Comment (cynepco3hahue) [inline core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:60]: operator-recovery -> operator-precache

Comment (vitus133) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay]: Done

**Pull request merged**